docs/api/spaces-management.asciidoc

```diff
@@ -1,16 +1,24 @@
 [role="xpack"]
 [[spaces-api]]
-== Kibana Spaces APIs
+== {kib} spaces APIs
 
 Manage your {kib} spaces.
 
-* <<spaces-api-post, Create space API>>
-* <<spaces-api-put, Update space API>>
-* <<spaces-api-get, Get a space API>>
-* <<spaces-api-get-all, Get all {kib} spaces API>>
-* <<spaces-api-delete, Delete space API>>
-* <<spaces-api-copy-saved-objects, Copy saved objects to space API>>
-* <<spaces-api-resolve-copy-saved-objects-conflicts, Resolve copy saved objects to space conflicts API>>
+The following {kib} spaces APIs are available:
```

Comment:
Some of intro pages have a lot of repetition between the title and the first two sentences.   I like the ones where you add a little bit more context in the first sentence.